Extensions/Settings/Dnn.PersonaBar.Connectors/Dnn.PersonaBar.Connectors.csproj

```diff
@@ -49,7 +49,7 @@
   <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.Targets" />
   <ItemGroup>
     <Reference Include="Dnn.PersonaBar.Library">
-      <HintPath>..\..\..\bin\Dnn.PersonaBar.Library.dll</HintPath>
```

Comment:
Is there a reason that this one (and _CssEditor_) remained a `Reference` instead of a `ProjectReference`?

Reply:
also _Licensing_, _Seo_, _Servers_, _SiteImportExport_, _SiteSettings_, _SqlConsole_, and _Vocabularies_

Reply:
no, must have missed them somehow